oracle_fdw.c

```diff
@@ -839,6 +894,10 @@ oracleGetForeignRelSize(PlannerInfo *root, RelOptInfo *baserel, Oid foreigntable
 	/* get Oracle's (bad) estimate only if plan_costs is set */
 	if (plan_costs)
 	{
+
+		fdwState->query = createQuery(fdwState, baserel, for_update, root->query_pathkeys);
+		elog(DEBUG1, "oracle_fdw: remote query is: %s", fdwState->query);
```

Comment:
It feels quite ugly to have to call `createQuery` here &ndash; but I understand the need.
I think the time has come to remove the `plan_costs` option. It has always been a wart, and it will remove one headache for us.
It has been deprecated for more than a year, so I think that should be OK.
Do you want to do that or shall I do it (including documentation)?